### ISSUE TITLE
minor glitch in Disassembly view

### ISSUE THREAD

**Issue by GitMensch** [KDAB/hotspot#339]
![contained &](https://user-images.githubusercontent.com/6699539/139075047-28f6168b-b08c-4b87-a6bd-473b8319a5aa.png)

the title has a contained `&`.

Hm, this isn't shown in the Dec2020 screenshot https://raw.githubusercontent.com/KDAB/hotspot/master/screenshots/dockwidgets.png is that a new issue or only happening on this machine?

**Comment by milianw** [comment]
no input since many months, closing

**Comment by GitMensch** [comment]
Wherever it is _recognized_ the acceleration key works fine (menu, "within the view", ...) but not for separate "windows"

![sample](https://user-images.githubusercontent.com/6699539/205068302-d62e5284-a345-45c7-8c3f-508d01df57fe.png)

**Comment by lievenhey** [comment]
I can confirm this issue on 5.26.3 with wayland
happens after moving a dockwidget in its own window
![grafik](https://user-images.githubusercontent.com/82457690/205939595-56912e24-2318-43a7-8e8d-77b65783cf4d.png)

**Comment by GitMensch** [comment]
So that's actually a QT issue? [note: the setting above does not change anything on the gitpod or local builds using x11 forwarding]

**Comment by iamsergio** [comment]
I think we reached the conclusion this was a Qt bug (not with all styles), with the implementation of drawing QStyle::CE_DockWidgetTitle

**Comment by GitMensch** [comment]
Thanks for the note - is there a workaround?

**Comment by iamsergio** [comment]
Bug should be reported to Qt. Here's a minimal repro without dock widgets

```
#include <QtWidgets>

// These work:
// ./qtwidgets_basic -style Breeze
// ./qtwidgets_basic -style Oxygen

// This one doesn't:
// ./qtwidgets_basic -style Fusion

class MyWidget : public QWidget {
public:
  void paintEvent(QPaintEvent *ev) override {
    QPainter p(this);

    QStyleOptionDockWidget titleOpt;
    titleOpt.initFrom(this);
    titleOpt.title = "&My Title";

    style()->drawControl(QStyle::CE_DockWidgetTitle, &titleOpt, &p, this);
  }
};

int main(int argc, char **argv) {
  QApplication app(argc, argv);

  qDebug() << QStyleFactory::keys();

  MyWidget window;

  window.show();

  return app.exec();
}
```

**Comment by GitMensch** [comment]
Thank you very much for inspecting this - I assume that is specific to (some) `QT_QPA_PLATFORM`?
Any clue if this is QT version specific?

**Comment by iamsergio** [comment]
It's specific to style, and not to QPA. I expect the same problem in all platforms where fusion is used.

No idea about AppImage, but that's plausible

**Comment by GitMensch** [comment]
Just a note, while waiting on possibly more styles to test with the appimage to have a workaround: This can also be produced with kcachegrind, so is totally a QT style issue.

Default style (same for `-style Windows`)  |  `-style Breeze`
----------------------------------------------|-----------------------
![default](https://user-images.githubusercontent.com/6699539/227663462-3555d7a9-108a-494d-ac17-198ecb25bafb.png) | ![breeze](https://user-images.githubusercontent.com/6699539/227663533-0d9e099b-35c7-49cc-bb8c-858c8ea2159e.png)

Can someone point me to the right issue tracker to report this "upstream" - or even better - to an existing ?
